eslint-example/package.json

```diff
@@ -0,0 +1,18 @@
+{
+    "name": "eslint-example",
+    "version": "1.0.0",
+    "description": "",
+    "main": "index.js",
+    "dependencies": {},
+    "devDependencies": {
+        "eslint": "^3.9.0",
+        "eslint-plugin-mozilla": "^0.2.3"
+    },
+    "scripts": {
+        "test": "echo \"Error: no test specified\" && exit 1",
+        "eslint": "./node_modules/.bin/eslint",
```

Comment:
This isn't necessary. The `npm run lint` command will already work fine as you have it because npm magically puts `./node_modules/.bin` on PATH for scripts.